### ISSUE TITLE
Speaker status incorrect

### ISSUE THREAD

**Issue by marktheknife** [dcmeglio/hubitat-heos#6]
When I have both my HEOS devices discovered, the HEOS 3 speaker always associates as the master, and there is plenty of log data from the device. The Marantz AVR doesn't show anything in the logs other than initializing telnet (but nothing else). Both device pages seem to get state data refreshed (like the name of the currently playing song), but the playing/stopped status never stays on "playing" even when the devices are streaming. In the logs it looks like the status updates briefly but then for some reason "stopped" is being reported again even though the music continues.

Here are some logs around the time I started a new stream from amazon music. First from the HEOS 3 speaker, second from the Marantz AVR, and third from the integration app.

![image](https://user-images.githubusercontent.com/16859844/73596466-42191000-44f0-11ea-9be1-b9f8f142cb86.png)

![image](https://user-images.githubusercontent.com/16859844/73596489-67a61980-44f0-11ea-872a-8013da55a7f7.png)

![image](https://user-images.githubusercontent.com/16859844/73596517-902e1380-44f0-11ea-874a-decb229ac219.png)

Some additional info that may or may not be helpful/relevant. The Marantz AVR is also integrated directly with hubitat using a Telnet-based driver, which was also true when I had only the AVR discovered as a HEOS device and did not notice the same issue with the playing state not showing correctly. The AVR and HEOS 3 are grouped together in the HEOS mobile app.

Thanks again for all your work on this!

**Comment by dcmeglio** [comment]
Hmm so first it is correct that the one device has no logs. Only the master device is connected to via telnet and is used to control all the devices. Are you saying you have two devices grouped together, both are playing, but the app reports that both are stopped? Looks like you're playing amazon music? Just trying to determine what you currently have it doing so I can try to reproduce it since it's been working for me. Is the play status correct if the devices aren't grouped?

**Comment by marktheknife** [comment]
Correct, when I start playing something from amazon music, both the AVR and the wireless speaker indeed play, and _some_ of the info in the current states of the device pages updates as it should. But the playing/stopped status seems like it always shows stopped, even while music is playing.

I will see what happens if I ungroup the HEOS devices and let you know.

**Comment by marktheknife** [comment]
After ungrouping the devices in the HEOS app, it looks like the status correctly updates now for each device in the hubitat web interface when I play/pause/stop music.

Here is a log from the HEOS 3 speaker when I start playing something on amazon and then pause it:

![image](https://user-images.githubusercontent.com/16859844/73610152-6fbe9180-45a2-11ea-88f4-9f14d2aaeb43.png)

When I use the HEOS app to start playing something from Amazon music on the Marantz AVR (and then stop it), here is the log from the HEOS 3 hubitat device:

![image](https://user-images.githubusercontent.com/16859844/73610231-1c007800-45a3-11ea-975b-4eeb10e567a1.png)

And the log from the HEOS integration app:

![image](https://user-images.githubusercontent.com/16859844/73610297-d1cbc680-45a3-11ea-800d-e85702867a9f.png)

I can do something similar when the devices are grouped in the HEOS app, if you think that's helpful.

**Comment by dcmeglio** [comment]
I've only managed to reproduce it once and it was right after grouping devices for the first time. As you see in the logs too, the device is definitely reporting a stop event. I'm not yet sure what's going on and I haven't found a consistent way to reproduce it yet.

**Comment by marktheknife** [comment]
I finally got a chance to keep playing around with this. It appears as though everything is updating as expected in the driver status page and the app logs. I've tried grouping and ungrouping the two speakers in the HEOS app a couple times, and I think it's still going strong.

When I get a chance I can take a screenshot of a few logs to see if there's still something floating around in there with those extra stop events.
